src/components/MovieCard/MovieCard.js

```diff
@@ -1,12 +1,21 @@
 import PropTypes from 'prop-types';
 import './MovieCard.css';
+import {Draggable} from 'react-beautiful-dnd'
 
-const Movie = ({ id, poster_path, title, enlargeCard, average_rating}) => {
+const Movie = ({ id, poster_path, title, enlargeCard, average_rating, index, isFav }) => {
   return (
-    <article className="movie-card" onClick={() => enlargeCard(id)}>
-      <img src={poster_path} alt={`${title} poster`}></img>
-      <p>Rating: {average_rating.toFixed(1)}</p>
-    </article>
+    <Draggable
+        draggableId={isFav? 'fav' + String(id) : String(id)}
+        index={index}>
+          {(provided) => ( <article className="movie-card" 
+          onClick={() => enlargeCard(id)}ref={provided.innerRef}
+          {...provided.draggableProps}
+          {...provided.dragHandleProps}>
+          <img src={poster_path} alt={`${title} poster`}></img>
+          <p>Rating: {average_rating.toFixed(1)}</p>
```

Comment:
I think the ratings on the enlarged page are fixed to 2. We should probably make it uniform to 1 or 2.